src/main/java/org/isf/permissions/manager/GroupPermissionManager.java

```diff
@@ -21,25 +21,30 @@
  */
 package org.isf.permissions.manager;
 
-import java.util.List;
-
 import org.isf.generaldata.MessageBundle;
 import org.isf.menu.model.UserGroup;
 import org.isf.permissions.model.GroupPermission;
 import org.isf.permissions.model.Permission;
 import org.isf.permissions.service.GroupPermissionIoOperations;
+import org.isf.permissions.service.PermissionIoOperations;
 import org.isf.utils.exception.OHDataValidationException;
 import org.isf.utils.exception.OHServiceException;
 import org.isf.utils.exception.model.OHExceptionMessage;
 import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
```

Comment:
wrong import order; please see the ide settings and adjust your IDE to always use them